app/Http/Controllers/UserController.php

```diff
@@ -3,58 +3,52 @@
 namespace App\Http\Controllers;
 
 use App\Models\User;
-use App\Exceptions\CustomException;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Http\RedirectResponse;
 use App\Services\Password\PasswordService;
-use App\Http\Requests\User\CreateUserRequest;
+use App\Http\Requests\User\UserDataRequest;
 use App\Http\Requests\User\UpdatePasswordRequest;
 use App\Services\Notification\NotificationService;
 
 class UserController extends Controller
 {
-    // Creating the dashboard for user  // ** REFACTORED
+    /**
+     * Return Index ** REFACTORED **
+     */
     public function index(User $user)
     {   
         $request = $user->findOrFail(Auth::id());
         return view('index', ['user' => $request]);
     }
-
-    // Store user data in database // ** REFACTORED
-    public function updateProfile(CreateUserRequest $request, User $user, NotificationService $notification)
+    /**
+     * Update Update Profile // ** REFACTORED
+     */
+    public function updateProfile(UserDataRequest $request, User $user, NotificationService $notification)
     {   
         // Find the user
-        $update = $user->findOrFail(Auth::id());
-
-        // Update user data
-        tap($update)->update($request->updateUserData())->save();
+        $user->findOrFail(Auth::id())->update($request->updateUserData());
         
         return redirect()->back()->with($notification->message('User Porfile Updated Successfully', 'success'));
     }
-
-    // User Logout // ** REFACTORED
-    public function destroy(NotificationService $notification): RedirectResponse 
+    /**
+     * User Logout // ** REFACTORED
+     */
+    public function logout(NotificationService $notification): RedirectResponse 
     {
         Auth::guard('web')->logout();
```

Comment:
`Auth::logout()` should work too. There shouldn't be any need to specify which guard to use.